templates/contribute/index.md

```diff
@@ -109,24 +109,37 @@ below the `---`.
-Everyone is also invited to regularly look at the queue (it is linkified as `#queue` on Zulip), and write reviews of PRs within their expertise.
+Everyone is also invited to regularly look at the queue (it is linkified as `#queueboard` on Zulip), and write reviews of PRs within their expertise.
+You can check if your [PR is on the queue](../queueboard/on_the_queue.html), and if not, what is needed to get it on.
 
 The review queue is controlled by GitHub "labels".
 On the main page for a PR, on the right-hand side,
 there should be a sidebar with panels "reviewers", "assignees", "labels", etc.
 Click on the "labels" header to add or remove labels from the current project.
 (Labels can only be edited by "GitHub collaborators", which is approximately the same as "people who have asked for write access".)
 
-If your PR builds (has a green checkmark), someone will probably "review" it within a few days (depending on the size of the PR; smaller PRs will get quicker responses). The reviewer will probably leave comments and add the label **"awaiting-author"**. You should address each comment, clicking the "resolve conversation" button once the problem is resolved. Ideally each problem is resolved with a new commit, but there is no hard rule here. Once all requested changes are implemented, you should remove the **"awaiting-author"** label to start the process over again.
-
-After some iteration, a maintainer or reviewer will "approve" the PR. (If it's a reviewer, this will add a "maintainer-merge" label, and soon after a maintainer will give final approval). This will result in a **"ready-to-merge"** label being automatically applied to the PR.
+If your PR builds (has a green checkmark), someone will "review" it within a few weeks (depending on the size of the PR; smaller PRs will get quicker responses). They will probably leave comments and add the label **"awaiting-author"**. You should address each comment, clicking the "resolve conversation" button once the problem is resolved. Ideally each problem is resolved with a new commit, but there is no hard rule here. Once all requested changes are implemented, you should remove the **"awaiting-author"** label to start the process over again.
+
+There are diffent groups of people that can review your PR: anyone, [reviewers](../teams/reviewers.html) and [maintainers](../teams/maintainers.html).
+Anyone who has something useful to say can review your PR. 
+If they think your PR is ready to move to the next stage, they might leave an "approving" review on GitHub. 
+These reviews are taken into account by reviewers. 
+If a reviewer considers your PR ready to be merged, they will add the **"maintainer-merge"** label to your PR. 
+These are used by maintainers to prioritize their review.
+Maintainers are always the ones to give final approval. 
+Depending on availability, a maintainer might look at your PR before a reviewer has. 
```

Comment:
Since maintainers are reviewers, this sentence as-written is a bit confusing. You mean to say something like the following, right? (Feel free to tweak the wording.)
```suggestion
Depending on availability, a maintainer could be the first reviewer to look at your PR: in this case,
your PR could get merged without being "maintainer merge"d first.